packages/amplication-git-service/src/utils/isInAmplicationIgnore.ts

```diff
@@ -0,0 +1,18 @@
+import minimatch from 'minimatch';
+
+export function isInAmplicationIgnore(
+  amplicationIgnoreStatements: string[],
+  filePath: string
+): boolean {
+  if (amplicationIgnoreStatements.length === 0) {
```

Comment:
If the amplication ignore statements can be of length 0, then we should not call the `isInAmplicationIgnore()` function to begin with. Let's move this check to `github.service.ts`.

Reply:
I agree and disagree...
I think you are right that the check needs to be the `github.service.ts` but I think the function still needs to be responsible of herself, what do you think?

Reply:
The `isInAmplicationIgnore` function accepts the "statements", which means the invoker already knows the statements. That means the invoker can make sure the statements have `length > 0`, so we remove the condition from this function and keep it simpler.

This function should already assume there is an `.amplicationignore` file and that it's not empty.